Specification_Sheet.tex

```diff
@@ -252,46 +252,47 @@ \subsection{the following data consistencies must be checked}
 \begin{center}
-\includegraphics[width=\textwidth]{generator}
+\includegraphics[width=\textwidth]{collector}
 \end{center}
 
-\subsubsection{Use of the labeling module}
-\textbf{Optional:} The user has a very specific problem which generates only a certain kind of sparse matrices. That is why he wants to adapt the neural network for his specific task. He furthermore only wants to use the default preconditioner/iterative solver combinations. He first of all safes all of his matrices in one directory of his choice. Then he opens the labeling module. The command label <filepath> will label all the matrices in the directory. The user may proceed with using the neural network.
+\subsubsection{Use of the \glspl{labeling module}}
+The user has a very specific problem which generates only a certain kind of sparse matrices. That is why he wants to adapt the neural network for his specific task. He furthermore only wants to use the default \glspl{preconditioner}/\glspl{iterative solver} combinations. He first of all safes all of his matrices in one directory x of his choice. Then he opens the \glspl{labeling module}. The command label -p x -n labeledMatrices will label all the matrices and generate their greyscale pattern image in the directory. The set of greyscale pattern images will be safed with the name labeledMatrices in the results file of the \glspl{labeling module}. The user may proceed with using the neural network.
 
 
-\textbf{Optional:} The user is fine with the default matrices, but he wants to use other preconditioner/iterative solver combinations which are included in the ginkgo libary. He opens the labeling module. With the command list combinations he will see all the combinations that are currently used. With the command add <preconditioner/iterative solver> a the new combinations of his choice will be added. With the command delete <preconditioner/iterative solver> the specified combination will be deleted. After the user made his choice he may proceed with using the neural network.
+\textbf{Optional:} The user is fine with the default matrices, but he wants to use other \glspl{preconditioner}/\glspl{iterative solver} combinations which are included in the ginkgo libary. He opens the \glspl{labeling module}. With the command list combinations he will see all the combinations that are currently used. With the command add <\glspl{preconditioner}/\glspl{iterative solver}> a the new combinations of his choice will be added. With the command delete <\glspl{preconditioner}/\glspl{iterative solver}> the specified combination will be deleted. After the user made his choice he may proceed with using the neural network.
```

Comment:
Im not quite sure if you would want to specify all the algorithims using the command line due to easy typo problems. I was thinking we might create a config file where we list all our standard solvers and the user can add and remove some on his own risk.